server/middleware/setUpEnvironmentName.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-param-reassign */
+import express from 'express'
+import config from '../config'
+
+export default (app: express.Express) => {
+  app.locals.environmentName = config.environmentName
+  app.locals.environmentNameColour = config.environmentName === 'PRE-PRODUCTION' ? 'govuk-tag--green' : ''
```

Comment:
Is this based on their instructions?
I'm not sure this works because it'll only use the green tag style if `config.environmentName === 'PRE-PRODUCTION'` .... but in `values-preprod.yml` it is set to `PREPROD`
So I dont think it'll ever get the green class ?

Reply:
Good spot - it is from the instructions specifically telling you to copy this file https://github.com/ministryofjustice/hmpps-micro-frontend-components/blob/main/_fallbacks/setUpEnvironmentName.ts.txt

I'll take a look and maybe open a PR/issue against their repo!